tardis/io/atom_data/base.py

```diff
@@ -223,35 +225,9 @@ def from_hdf(cls, fname=None):
 
             atom_data = cls(**dataframes)
 
-            try:
-                atom_data.uuid1 = store.root._v_attrs["uuid1"]
-                if hasattr(atom_data.uuid1, "decode"):
-                    atom_data.uuid1 = store.root._v_attrs["uuid1"].decode(
-                        "ascii"
-                    )
-            except KeyError:
-                logger.debug(
-                    "UUID not available for Atom Data. Setting value to None"
-                )
-                atom_data.uuid1 = None
-
-            try:
-                atom_data.md5 = store.root._v_attrs["md5"]
-                if hasattr(atom_data.md5, "decode"):
-                    atom_data.md5 = store.root._v_attrs["md5"].decode("ascii")
-            except KeyError:
-                logger.debug(
-                    "MD5 not available for Atom Data. Setting value to None"
-                )
-                atom_data.md5 = None
-
-            try:
-                atom_data.version = store.root._v_attrs["database_version"]
-            except KeyError:
-                logger.debug(
-                    "VERSION not available for Atom Data. Setting value to None"
-                )
-                atom_data.version = None
+            set_atom_data_attributes(atom_data, store, "uuid1")
```

Comment:
Why not make this a method or an operator overload?

Reply:
I was simply refactoring the try-except blocks out. If there's a more pythonic way, I can modify this.

Reply:
I think the question was why we don't have this as a method as part of the class.